service/src/main/java/uk/gov/hmcts/reform/fpl/validators/HasThresholdCriteriaValidator.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import uk.gov.hmcts.reform.fpl.enums.OrderType;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasThresholdCriteria;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class HasThresholdCriteriaValidator implements ConstraintValidator<HasThresholdCriteria, CaseData> {
```

Comment:
Can we differentiate this from HasThresholdCriteriaDetailsValidator a little more please. The first does ThresholdDetails, this does ThresholdReason